docs/section21/Rule_21-2.md

```diff
@@ -0,0 +1,39 @@
+
+# Boiler - Rule 21-2  
+
+**Rule ID:** 21-2  
+**Rule Description:** For purchased HW/steam in the proposed model, the baseline shall have the same number of pumps as proposed
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.1.1 & G3.1.1.3.1 Building System-Specific Modeling Requirements for the Baseline model  
+**Schema Version:** 0.0.25
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. P-RMR is modeled with purchased hot water or steam.
+
+**Manual Check:** Yes  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** 
+
+1. check_purchased_chw_hhw()
+
+
+**Applicability Checks:**
+
+- Check if P-RMR is modeled with purchased chilled water or purchased hot water/steam: `purchased_chw_hhw_status_dict = check_purchased_chw_hhw(P_RMR)`
+
```

Comment:
this rule is applicable is P RMDi is modeled with PCHW or PHW/Steam. linke 14 only mentions PHW/Steam and like 26 mentions all 3 but the check is being done for PHW and Steam only.
should consistently check for purchased heating and cooling.

Reply:
section 21 consistently deals only with heating systems.  I believe there should be an equivalent rule in section 21 that deals with purchased cooling